test/jdk/javax/net/ssl/TLSCommon/TLSTest.java

```diff
@@ -51,6 +57,7 @@
  * @bug 8205111
  * @summary Test TLS with different types of supported keys.
  * @run main/othervm TLSTest TLSv1.3 rsa_pkcs1_sha1 TLS_AES_128_GCM_SHA256
+ * @run main/othervm -Djdk.tls.server.SignatureSchemes=ecdsa_brainpoolP512r1tls13_sha512 -Djdk.tls.client.SignatureSchemes=ecdsa_brainpoolP512r1tls13_sha512 -Djdk.tls.namedGroups=brainpoolP512r1tls13 TLSTest TLSv1.3 ecdsa_brainpoolP512r1_sha512 TLS_AES_128_GCM_SHA256
```

Comment:
Suggest folding this (see folded examples below) and defining properties alphabetically:
```
 * @run main/othervm
 *      -Djdk.tls.client.SignatureSchemes=ecdsa_brainpoolP512r1tls13_sha512
 *      -Djdk.tls.namedGroups=brainpoolP512r1tls13
 *      -Djdk.tls.server.SignatureSchemes=ecdsa_brainpoolP512r1tls13_sha512
 *      TLSTest TLSv1.3 ecdsa_brainpoolP512r1_sha512 TLS_AES_128_GCM_SHA256
```

Reply:
Folded this line as suggested and alphabetized the arguments to the test.